app/_kong_plugins/openid-connect/examples/mtls-client-auth.yaml

```
@@ -1,36 +1,45 @@
 title: Mutual TLS client authentication
 description: |
-  Configure the OpenID Connect plugin to use mutual TLS (mTLS) client authentication.
+  Configure the OpenID Connect plugin to use [mutual TLS (mTLS) client authentication](/plugins/openid-connect/#mutual-tls-client-authentication).
 
   The following uses the password grant, but you can use any supported OpenID Connect auth method.
 
+  The configuration option `config.tls_client_auth_ssl_verify` controls whether the server (IdP) certificate is verified.
+  When set to `true` (default), ensure that [trusted certificate](/gateway/configuration/#lua-ssl-trusted-certificate) and [verify depth](/gateway/configuration/#lua-ssl-verify-depth) are appropriately configured so that the IdP's server certificate is trusted by {{site.base_gateway}}.
+
 weight: 819
 
 requirements:
   - A configured identity provider (IdP) configured with mTLS and X.509 client certificate authentication
   - A client certificate and key pair stored in a [Certificate object](/gateway/entities/certificate/)
 
 config:
-  issuer: $ISSUER
+  issuer: ${issuer}
```

```suggestion
  issuer: http://example-keycloak-host:8080/realms/example-realm
```
This example wouldn't work for me unless I didn't use the env var.